src/pages/TypesFinancialInstitutions/index.tsx

```diff
@@ -135,7 +135,7 @@ function TypesFinancialInstitutions(): JSX.Element {
           alertHeading={
             hasOtherFormError
               ? 'There was a problem updating your type of financial institution'
-              : 'You must select your type of financial institution to save and continue'
+              : 'You must select your type of financial institution to continue to next step'
```

Comment:
Update: 
'There was a problem updating your type of financial institution'

Reply:
Heyo @natalia-fitzgerald, this change was already made when your review went up. GitHub will show you a "refresh" option while you're doing your review.